lib/client/api.go

```diff
@@ -510,7 +510,8 @@ func Status(profileDir string, proxyHost string) (*ProfileStatus, []*ProfileStat
 	profileDir = FullProfilePath(profileDir)
 	stat, err := os.Stat(profileDir)
 	if err != nil {
-		return nil, nil, trace.Wrap(err)
+		log.Debugf("Failed to stat file: %v.", err)
+		return nil, nil, trace.NotFound(err.Error())
```

Comment:
We should only return `trace.NotFound` when `os.IsNotExist(err)`.
Otherwise it could be e.g. a permission issue, which should get surfaced